storage/backend/azure/azure.go

```diff
@@ -46,8 +46,13 @@ func New(l log.Logger, c Config) (*Backend, error) {
 	// 3. Azurite has different URL pattern than production Azure Blob Storage.
 	var blobURL *url.URL
 	if c.Azurite {
+		fmt.Println("Container Name: %s", c.ContainerName)
```

Comment:
this will result into printing `Container Name: %s blah`
do` fmt.Println("Container Name: ", c.ContainerName)`

Reply:
I have made the changes. Thank you for noticing!